packages/vuepress-theme-odyssey/global-components/Description.vue

```diff
@@ -0,0 +1,11 @@
+<template>
+  <div class="docskit--desc">
+    <slot />
```

Comment:
nit: Try to use closing tags instead of self-closing. This is a practice we try to encourage to avoid known scripting vulns.

Reply:
This is good feedback, will fix.